apps/smtp-server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "smtp-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "start": "node dist/server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/dotenv": "^8.2.0",
+    "@types/mailparser": "^3.4.4",
+    "@types/smtp-server": "^3.5.10",
+    "axios": "^1.7.3",
```

Comment:
do we really need this? we can just use fetch right?

Reply:
It is required for Types definition for Javascript libraries. We don't use this in NextJs because NextJs usually handles this for us.